### PR TITLE
Version 1.34.0 → 1.35.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

File: dhall-bash/dhall-bash.cabal [hunk 1]
```diff
@@ -31,7 +31,7 @@ Library
         base                      >= 4.11.0.0 && < 5   ,
         bytestring                               < 0.11,
         containers                               < 0.7 ,
-        dhall                     >= 1.34.0   && < 1.35,
+        dhall                     >= 1.34.0   && < 1.36,
```
Comment (german1608) [inline dhall-bash/dhall-bash.cabal:34]: Shouldn't the lower bound be `1.35.0`?

File: dhall-nixpkgs/dhall-nixpkgs.cabal [hunk 2]
```diff
@@ -19,7 +19,7 @@ Executable dhall-to-nixpkgs
   Build-Depends:       base                 >= 4.11     && < 5
                      , aeson                >= 1.0.0.0  && < 1.5
                      , data-fix
-                     , dhall                >= 1.32.0   && < 1.35
+                     , dhall                >= 1.32.0   && < 1.36
```
Comment (german1608) [inline dhall-nixpkgs/dhall-nixpkgs.cabal:22]: Same comment here. Shouldn't the lower bound be `1.35.0`?

**Comment by Gabriel439** [comment]
@german1608: The reason why not all packages change the lower bound is that some of them were unaffected by the breaking changes introduced in version 1.35.0, so as far as I can tell those packages still build against version 1.34.0.

**Comment by german1608** [comment]
@Gabriel439 alright, makes sense!

**Comment by Gabriel439** [comment]
Sorry for the delay in cutting this.  I just needed to fix https://github.com/dhall-lang/dhall-haskell/pull/2030 first since it's sufficiently disruptive

**Review by sjakobi — APPROVED** [review]

**Review by sjakobi — APPROVED** [review]

**Pull request merged**